packages/front-end/components/Tags/TagsInput.tsx

```diff
@@ -86,16 +86,16 @@ const TagsInput: FC<{
         ...styles,
         borderRadius: 4,
         backgroundColor: data.color,
-        color: useDarkText(data.color) ? "#000000" : "#ffffff",
+        color: useDarkText(data.color) ? "#000000" : "#ffffff", // eslint-disable-line
```

Comment:
The linter was really unhappy with this because the name of this function doesn't start with a capital letter. It felt really weird to add the capital letter here, so I just disabled the line, but curious if you think there is a better solution here. Given that it relies on `data.color` I can't think of a way to evaluate the expression of `useDarkTextColor(data.color)` outside of this object.

Reply:
We should probably change the name of the `useDarkText` function.  Functions should only start with `use` if they are a hook and this function is just a tiny util function we made without any dependencies.  Some possible alternate names: `needsDarkText`, `shouldUseDarkText`, `isLightBackground`.

Reply:
Ah, makes sense - I didn't realize `useDarkText` wasn't actually a hook. Updated to `isLight`.